_topic_maps/_topic_map_ms.yml

```diff
@@ -33,14 +33,14 @@ Name: Release notes
 Dir: microshift_release_notes
 Distros: microshift
 Topics:
-- Name: MicroShift 4.14 release notes
+- Name: Red Hat build of MicroShift 4.14 release notes
   File: microshift-4-14-release-notes
 ---
 Name: Getting started
 Dir: microshift_getting_started
 Distros: microshift
 Topics:
-- Name: Understanding MicroShift
+- Name: Understanding Red Hat build of MicroShift
```

Comment:
I feel compelled to add a "the" before Red Hat build, but I guess this may be tricky brand/legal-wise...just putting this 💭 out there.

Reply:
@skrthomas Agree, but most of the other 'RH build of...' products on the portal don't use the article.